### DISCUSSION TITLE
Any way to jump between Trouble list items without opening the Trouble window?

### DISCUSSION THREAD

**Discussion by Hubro** [folke/trouble.nvim#376] [Q&A]
I often open quickfix items or LSP diagnostics in Trouble, but the window itself takes up quite a bit of room. I often want to initially open the items in Trouble just to gauge the scope of what I have to do, then go to the first item, closing the Trouble panel. I now miss the ability to jump to the next item in Trouble without opening the panel.

Is this currently possible? `:lua require("trouble").next({ jump = true })` crashes if the Trouble window isn't open.

---

I know I can use `:cnext` and `:cprevious` when I'm working on quickfix items, but this goes through the items in a different order than the Trouble window, which is confusing. It also doesn't apply to other lists opened in Trouble.

**Comment by Demianeen** [comment]
As far as I understand, this is because trouble can open different windows. In lazyvim trouble can be used like quickfix list for telescope for example.

It probably introduces some complexity and can be unintuitive what trouble list to use when it is closed

**Reply by Hubro** [reply]
It think it would be nice if Trouble remembered that anyway, much like `:Telescope resume` which I use constantly.

**Reply by Demianeen** [reply]
You can do pr if you want, it does not sounds hard. You just have table that will contain history state. So when you do `:Trouble resume`. It just uses last table item

**Comment by RiteshChepuri** [comment]
Did you try anything regarding this?
